src/js/utils/index.js

```diff
@@ -17,6 +17,8 @@ import {
 import { startOfWeek, endOfWeek } from "date-fns"
 import { format } from "date-fns"
 
+export const globalBrowserObject = () => globalThis.browser ?? globalThis.chrome
```

Comment:
There a issues in Firefox using `globalThis.chrome`, e.g. in `chrome.storage.sync.get`. That's why I introduced this function. `globalThis.browser` is not defined in Chrome. `globalThis` is `window` on the client, and the global object in the service worker.